format/spec.md

```diff
@@ -44,6 +44,15 @@ The primary change in version 2 adds delete files to encode rows that are delete
 
 In addition to row-level deletes, version 2 makes some requirements stricter for writers. The full set of changes are listed in [Appendix E](#version-2).
 
+#### Version 3: Extended Types and Capabilities
+
+Version 3 of the Iceberg spec extends data types and existing metadata structures to add new capabilities, including:
+
+* Adding new data types, like nanosecond timestamps, variant, and unknown.
+* Adding new supported type promotion cases, integer and long to string
+* Adding default value support to struct fields
```

Comment:
As an aside, I think the default value wording in the spec is a little strange, as the way I read it, it allows for default values of lists and maps as long as they are enclosed in a struct which seems like an odd restriction that at least warrants some further explanations.

Reply:
Both lists/maps inside of a struct, and the list/map itself can have a default value, see https://github.com/apache/iceberg/blob/main/format/spec.md#schemas

Or if you want to get your hands dirty https://github.com/apache/iceberg/pull/9502/

Reply:
@emkornfield can we discuss the default value behavior separately? For this PR, I just want to call out that it was added and is required in v3.

Reply:
I agree, I guess would maybe simplify the language here to simply be adding "default value support for columns".